crates/subspace-farmer/src/utils/archival_storage_info.rs

```diff
@@ -13,18 +13,18 @@ use subspace_networking::libp2p::PeerId;
 use subspace_networking::CuckooFilterDTO;
 
 const CONNECTED_PEERS_NUMBER_LIMIT: usize = 50;
-const DISCONNECTED_PEERS_NUMBER_LIMIT: usize = 50;
+
+struct PeerFilter {
+    cuckoo_filter: CuckooFilter<DefaultHasher>,
+    // defines whether the peer owning this filter is currently connected
+    peer_connected: bool,
```

Comment:
I thought we agreed we'll remove connection status and drop filters on disconnection to keep the logic simple.